src/main/java/org/broadinstitute/consent/http/service/DatasetRegistrationService.java

```diff
@@ -211,6 +214,15 @@ public List<Dataset> createDatasetsFromRegistration(
     } catch (Exception e) {
       logException(e);
     }
+    try (Response response = openSearchService.indexDatasets(createdDatasetIds, user)) {
+      if (response.getStatus() >= 400) {
+        logWarn(
+            String.format(
+                "Error indexing datasets from registration: %s", registration.getStudyName()));
```

Comment:
I like the `response.getStatus() >= 400` construct.  

I see it's logged as a warning, but the message says error.  I also see this log-error-via-warning is pretty common here.  If a client (4xx) or server (5xx) error from `indexDatasets` is generally ignorable, this seems reasonable even if slightly discrepant.